### PR TITLE
suggestions for aws backup restore

### PR DESCRIPTION
**Pull request by leena-jawale**

### CODE REVIEW

**Review by zkayyali812 — APPROVED** [review]

File: stories/topics/proc-gcp-deploying-extension-nodes.adoc [hunk 1]
```diff
@@ -10,6 +10,7 @@ You configure extension nodes after you have purchased and launched an extension
 . In the *Region* field, select the region where you want the application deployed.
 . In the *Zone* field, select a zone in the *Region* you selected when deploying your foundational offer. 
 This field is only used to filter the Network list.
+. In the *Main Deployment Name* field, enter the foundation deployment name for which you are deploying extension.
```
Comment (ianf77) [inline stories/topics/proc-gcp-deploying-extension-nodes.adoc:13]: deploying an extension.

**Review by ianf77 — COMMENTED** [review]
One minor comment

**Comment by chrisahl** [comment]
@ianf77 Please take a look at the update.  Thanks

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**